modules/telemetry/telemetry.py

```diff
@@ -46,15 +39,14 @@
 logger = logging.getLogger(__name__)
 
 
-# Signal handler
 def shutdown_sequence() -> None:
```

Comment:
Not sure how this happened but the changes to the function signatures of `shutdown_sequence` didn't go through, it should look like
`def shutdown_sequence(signum: int, stack_frame: FrameType):`

Reply:
Fixing!